### ISSUE TITLE
problem with Autograd

### ISSUE THREAD

**Issue by dengmj13246817632** [pymanopt/pymanopt#278]
I have a problem with the python library pymanopt.
I don’t know why this line doesn’t work on my machine:
" @pymanopt.function.Autograd(manifold) "
The error says that Backend 'Autograd' is not available and I am using the latest version of the library 2.2.0.
I used the same code as the one on the website https://pymanopt.org/docs/latest/quickstart.html

**Comment by NicolasBoumal** [comment]
I had some issues recently too, using Python 3.12 on Windows.
The following sequence of pip commands worked out on my machine (fresh Python install), so that the 'quick start' example executes as intended:

```
pip install scipy
pip install autograd
pip install pymanopt --no-dependencies
```

I'd be interested to know if this workaround works for you too, or if you resolved things a different way.

**Comment by NicolasBoumal** [comment]
This should be fixed with release 2.2.1. As now instructed on the quick start page, install with:
`pip install "pymanopt[autograd]"`
This will make sure autograd is also available on your machine.
